URA-695/cell_line_analysis.py

```
@@ -0,0 +1,258 @@
+from cProfile import label
+import pandas as pd
+from scipy.stats import pearsonr
+from correlation_plots import transformed_vcf, merge_vcf_common_sites
+import os
+import seaborn as sns
+import matplotlib.pyplot as plt
+import numpy as np
+from plotnine import *
```

Ruff (0.8.0)

`from plotnine import *` used; unable to detect undefined names

(F403)